dev/reference/configuration/tidb-server/configuration-file.md

```diff
@@ -66,6 +66,12 @@ The TiDB configuration file supports more options than command-line parameters.
 - Determines whether to treat the `utf8` character set in old tables as `utf8mb4`.
 - Default value: `true`
 
+### `alter-primary-key`
+
+- Determine whether to add or remove the `primary key`.
```

Comment:
```suggestion
- Determines whether to add or remove the primary key constraint to or from a column.
```
1. To unify the language styles in this document.
2. No need to wrap "primary key" with backticks.

Plus, I added some details to the sentence. Please confirm with the author about whether it's correct. @toutdesuite @zimulala